proposals/simd/BinarySIMD.md

```diff
@@ -98,67 +98,67 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
-| `f64x2.sqrt`              |    `0xa8`| -                  |
+| `i8x16.neg`               |    `0x51`| -                  |
+| `i8x16.any_true`          |    `0x52`| -                  |
+| `i8x16.all_true`          |    `0x53`| -                  |
+| `i8x16.add`               |    `0x54`| -                  |
+| `i8x16.add_saturate_s`    |    `0x55`| -                  |
+| `i8x16.add_saturate_u`    |    `0x56`| -                  |
+| `i8x16.sub`               |    `0x57`| -                  |
+| `i8x16.sub_saturate_s`    |    `0x58`| -                  |
+| `i8x16.sub_saturate_u`    |    `0x59`| -                  |
+| `i8x16.mul`               |    `0x5a`| -                  |
+| `i8x16.shl`               |    `0x5f`| -                  |
+| `i8x16.shr_s`             |    `0x60`| -                  |
+| `i8x16.shr_u`             |    `0x61`| -                  |
+| `i16x8.neg`               |    `0x62`| -                  |
+| `i16x8.any_true`          |    `0x63`| -                  |
+| `i16x8.all_true`          |    `0x64`| -                  |
+| `i16x8.add`               |    `0x65`| -                  |
+| `i16x8.add_saturate_s`    |    `0x66`| -                  |
+| `i16x8.add_saturate_u`    |    `0x67`| -                  |
+| `i16x8.sub`               |    `0x68`| -                  |
+| `i16x8.sub_saturate_s`    |    `0x69`| -                  |
+| `i16x8.sub_saturate_u`    |    `0x6a`| -                  |
+| `i16x8.mul`               |    `0x6b`| -                  |
+| `i16x8.shl`               |    `0x70`| -                  |
+| `i16x8.shr_s`             |    `0x71`| -                  |
+| `i16x8.shr_u`             |    `0x72`| -                  |
+| `i32x4.neg`               |    `0x73`| -                  |
+| `i32x4.any_true`          |    `0x74`| -                  |
+| `i32x4.all_true`          |    `0x75`| -                  |
+| `i32x4.add`               |    `0x76`| -                  |
+| `i32x4.sub`               |    `0x79`| -                  |
+| `i32x4.mul`               |    `0x7c`| -                  |
+| `i32x4.shl`               |    `0x81`| -                  |
+| `i32x4.shr_s`             |    `0x82`| -                  |
+| `i32x4.shr_u`             |    `0x83`| -                  |
+| `i64x2.neg`               |    `0x84`| -                  |
+| `i64x2.any_true`          |    `0x85`| -                  |
+| `i64x2.all_true`          |    `0x86`| -                  |
+| `i64x2.add`               |    `0x87`| -                  |
+| `i64x2.sub`               |    `0x8a`| -                  |
+| `i64x2.shl`               |    `0x92`| -                  |
+| `i64x2.shr_s`             |    `0x93`| -                  |
+| `i64x2.shr_u`             |    `0x94`| -                  |
+| `f32x4.abs`               |    `0x95`| -                  |
+| `f32x4.neg`               |    `0x96`| -                  |
+| `f32x4.sqrt`              |    `0x97`| -                  |
```

Comment:
what's the gap here for?

Reply:
Experimentation with reciprocal sqrt and approximate reciprocal, as recommended by @dtig.

Reply:
Ah, cool. Sounds good!